stdlib/src/locale.rs

```diff
@@ -0,0 +1,101 @@
+#[cfg(unix)]
+pub(crate) use _locale::make_module;
+
+#[cfg(unix)]
+#[pymodule]
+mod _locale {
+    use rustpython_vm::{
+        builtins::{PyDictRef, PyIntRef, PyListRef},
+        PyObjectRef, PyResult, VirtualMachine,
+    };
+
+    #[pyattr]
+    use libc::{
+        ABDAY_1, ABDAY_2, ABDAY_3, ABDAY_4, ABDAY_5, ABDAY_6, ABDAY_7, ABMON_1, ABMON_10, ABMON_11,
+        ABMON_12, ABMON_2, ABMON_3, ABMON_4, ABMON_5, ABMON_6, ABMON_7, ABMON_8, ABMON_9,
+        ALT_DIGITS, AM_STR, CODESET, CRNCYSTR, DAY_1, DAY_2, DAY_3, DAY_4, DAY_5, DAY_6, DAY_7,
+        D_FMT, D_T_FMT, ERA, ERA_D_FMT, ERA_D_T_FMT, ERA_T_FMT, LC_ALL, LC_COLLATE, LC_CTYPE,
+        LC_MESSAGES, LC_MONETARY, LC_NUMERIC, LC_TIME, MON_1, MON_10, MON_11, MON_12, MON_2, MON_3,
+        MON_4, MON_5, MON_6, MON_7, MON_8, MON_9, NOEXPR, PM_STR, RADIXCHAR, THOUSEP, T_FMT,
+        T_FMT_AMPM, YESEXPR,
+    };
+
+    use std::ffi::{c_char, CStr};
+
+    #[pyattr(name = "CHAR_MAX")]
+    fn char_max(vm: &VirtualMachine) -> PyIntRef {
+        vm.ctx.new_int(libc::c_char::MAX)
+    }
+
+    unsafe fn copy_grouping(group: *mut c_char, vm: &VirtualMachine) -> PyListRef {
+        let mut group_vec: Vec<PyObjectRef> = Vec::new();
+        let mut ptr = group;
+
+        while ![0 as i8, c_char::MAX].contains(&*ptr) {
```

Comment:
@youknowone Clippy wants this:

```suggestion
        while ![0_i8, c_char::MAX].contains(&*ptr) {
```